pkg/agent/openflow/network_policy.go

```diff
@@ -991,6 +999,49 @@ func (c *client) InstallPolicyRuleFlows(rule *types.PolicyRule) error {
 	return nil
 }
 
+// Check is the rule is egress for IGMP, caller will return empty flow if it is.
```

Comment:
Maybe put some comments before these funcs, saying: For multicast traffic, we support ingress rules for IGMP, and egress rules for multicast data traffic. Ingress rules for IGMP packets are enforced by the packetIn handler.